src/stan/services/util/create_unit_e_dense_inv_metric.hpp

```diff
@@ -17,17 +17,12 @@ namespace util {
  * @return var_context
  */
 inline stan::io::dump create_unit_e_dense_inv_metric(size_t num_params) {
-  Eigen::MatrixXd inv_metric(num_params, num_params);
-  inv_metric.setIdentity();
-  size_t num_elements = num_params * num_params;
+  auto num_params_str = std::to_string(num_params);
+  std::string dims("),.Dim=c(" + num_params_str + ", " + num_params_str + "))");
+  Eigen::IOFormat RFmt(Eigen::StreamPrecision, Eigen::DontAlignCols, ", ", ",",
+                       "", "", "inv_metric <- structure(c(", dims);
   std::stringstream txt;
-  txt << "inv_metric <- structure(c(";
-  for (size_t i = 0; i < num_elements; i++) {
-    txt << inv_metric(i);
-    if (i < num_elements - 1)
-      txt << ", ";
-  }
-  txt << "),.Dim=c(" << num_params << ", " << num_params << "))";
+  txt << Eigen::MatrixXd::Identity(num_params, num_params).format(RFmt);
```

Comment:
These changes seem unrelated. Nothing wrong with quality of life changes, but it does make it harder to review these pulls cause I gotta check these side things.

Reply:
Yeah sorry about that, was just looking through the function we use here and saw this one could be cleaned up a little bit. I'll avoid this in the future